go/install/install_osx.go

```diff
@@ -161,6 +161,7 @@ func ListServices(g *libkb.GlobalContext) (*keybase1.ServicesStatus, error) {
 func DefaultLaunchdEnvVars(g *libkb.GlobalContext, label string) []launchd.EnvVar {
 	return []launchd.EnvVar{
 		launchd.NewEnvVar("KEYBASE_LABEL", label),
+		launchd.NewEnvVar("KEYBASE_SERVICE_TYPE", "launchd"), // This was added 03/22/16
```

Comment:
can you change the check in `service/main` to refer to this rather than `KEYBASE_LABEL`?